docs/Explanation/ADR/adr-004-did.md

```diff
@@ -89,7 +90,7 @@ To create and publish a DID document use the message
 MsgCreateDidDocument(id string, signerPubKey string)
 ```
 
-The message parameters are the DID to be created and the `signerPubKey`. The `signerPubKey` MUST be the public key of the account that signs the transaction. The public key MUST be used to attach a verification method of type `EcdsaSecp256k1VerificationKey2019` with the value of `publicKeyHex` that contains the public key encoded in hexadecimal.
+The message parameters are the DID to be created and the `signerPubKey`. The `signerPubKey` MUST be the public key of the account that signs the transaction. The public key MUST be used to attach a verification method of type `EcdsaSecp256k1VerificationKey2019` with the value of `publicKeyMultibase` that contains the public key encoded according to the [hexadecimal multibase format](https://datatracker.ietf.org/doc/html/draft-multiformats-multibase#appendix-B.1).
```

Comment:
```suggestion
The message parameters are the DID to be created and the `signerPubKey`. The `signerPubKey` MUST be the public key of the account that signs the transaction. The public key MUST be used to attach a verification method of type `EcdsaSecp256k1VerificationKey2019` with the value of `publicKeyMultibase` that contains the public key encoded according to the Multibase Data Format [hexadecimal Hexadecimal upper-case encoding](https://datatracker.ietf.org/doc/html/draft-multiformats-multibase#appendix-B.1).
```

Reply:
oh, for got to save this suggestion on first review... the goal is to make link text descriptive, so using the title of the section for the destination page is a better signpost for our readers